tests/serverpod_test_server/test_e2e/connection_test.dart

```diff
@@ -818,6 +820,16 @@ void main() {
       var result = await client.basicTypes.testUuid(null);
       expect(result, isNull);
     });
+
+    test('Type Uri', () async {
+      var result = await client.basicTypes.testUri(uri);
+      expect(result, equals(uri));
+    });
```

Comment:
Suggestion: Add the same test in the file `test/serialization_test.dart`

Reply:
And while we are at it, lets break out these type e2e tests into a standalone file and update the test description. There are a lot of old tests in here I can see.

I think you can move the URI setup into the test too, instead of having it declared so far up in the code. Then it becomes easier to see what we are testing.

Reply:
> Suggestion: Add the same test in the file test/serialization_test.dart

Are you referring to
https://github.com/tp/serverpod/blob/f6b6b8983e50152f697fccbc897b994353cd5ae5/tests/serverpod_test_server/test/serialization_test.dart#L102

I wanted to add the support for "in model" later with the other ticket, but I could also just add it now.

Looking at places where for example UUID is tested, I saw though that I was missing one in https://github.com/tp/serverpod/blob/f6b6b8983e50152f697fccbc897b994353cd5ae5/packages/serverpod_serialization/test/decode_with_type_test.dart#L251 that I have just added.

Also moved the basic type tests into their own file and aligned the descriptions with the pattern.

Reply:
Ahh you are right! Sorry I was getting a head of my self there! 

But good find on the other test file!